draft-ietf-tls-wkech.xml

```diff
@@ -72,12 +72,16 @@
         <t>
             We define a well-known URI at which an HTTP origin can
             inform an authoritative DNS server, or other interested parties,
-            about this origin's Service Bindings, i.e. its "HTTPS" DNS records.
-            These instructions can include Encrypted ClientHello (ECH)
+            about this origin's Service Bindings, such as its "HTTPS" DNS records.
+            This data can include Encrypted ClientHello (ECH)
```

Comment:
"data" is plural :)

Reply:
Language is evolving :)  But sure, "This data" -> "The data" in the fixup commit.